spring-cloud-dataflow-configuration-metadata/src/main/resources/META-INF/dataflow.properties

```diff
@@ -0,0 +1 @@
+configuration.properties = server.port
```

Comment:
This ends up being a global default.... I don't think we want to have `server.port` listed for every app.